website/source/docs/acl/acl-rules.html.md.erb

```diff
@@ -561,3 +561,69 @@ session "admin" {
 Session rules are segmented by the node name they apply to. In the example above, the rules allow read-only
 access to sessions on node name with the empty prefix, allow creating sessions on the node named "app",
 and deny all access to any sessions on the "admin" node.
+
+#### Namespacing
+
+<%= enterprise_alert :consul %>
+
+[Consul Enterprise](https://www.hashicorp.com/consul.html) 1.7.0 adds support for Namespacing of many 
+Consul resources. Rules can then be defined to only to apply to apply to specific namespaces.
+
+A Namespace specific rule would look like this:
+
+```hcl
+namespace_prefix "" {
+  
+  # grant service:read for all services in all namespaces
+  service_prefix "" {
+    policy = "read"
+  }
+  
+  # grant node:read for all nodes in all namespaces
+  node_prefix "" {
+    policy = "read"
+  }
+}
+
+namespace "foo" {
+  # grants permission to manage ACLs only for the foo namespace
+  acl = "write"
+  
+  # grants write permissions to the KV for namespace foo
+  key_prefix "" {
+    policy = "write"
+  }
+  
+  # grants write permissions for sessions for namespace foo
+  session_prefix "" {
+    policy = "write"
+  }
+  
+  # grants service:write for all services in the foo namespace
+  service_prefix "" {
+    policy = "write"
+  }
+  
+  # grants node:read for all nodes
+  node_prefix "" {
+    policy = "read"
+  }
+}
+```
+
+There are some restrictions to the rules that can be contained within a `namespace` or `namespace_prefix` rule that
```

Comment:
```suggestion
There are some restrictions to the rules that can be contained within a `namespace` or `namespace_prefix` rule. Generally, rules must
```

Reply:
What I am trying to convey here is that:

This is okay:

```
namespace "default" {
   operator = "write"
}
```

But this is not:

```
namespace "foo" {
   operator = "write"
}
```

so I was trying to say that when the namespace rule's segment is not `default` the extra restrictions kick in.

Reply:
```
Note, when a rule is defined in any user created namespace, the following restrictions apply. 
1.
1.
These restrictions do not apply to the `default` namespace created by Consul. 
```
How does that sound?

Reply:
Sounds great.